config/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const webpack = require("webpack");
+const path = require("path");
+const BabiliPlugin = require("babili-webpack-plugin");
+const CleanWebpackPlugin = require("clean-webpack-plugin");
+
+module.exports = {
+  entry: path.resolve(__dirname, "../index.js"),
```

Comment:
seems like you don't need path if you set context?

https://webpack.github.io/docs/configuration.html